src/main/app/src/app/csv/csv-export-button/csv-export-button.component.html

```diff
@@ -0,0 +1,9 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2022 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
+<button (click)="downloadCsv()" class="mat-paginator" id="CSV_button" mat-button
```

Comment:
Ik zou overal het woord csv verwijderen en zoiets als downloadExport doen. Ook het component hernoemen naar export-button.component

Reply:
Gerefactored in nieuwe commit